src/sentry/static/sentry/app/components/events/interfaces/threads/threadSelector/getThreadException.tsx

```diff
@@ -2,27 +2,33 @@ import {Thread} from 'app/types/events';
 import {Event, ExceptionType} from 'app/types';
 import {defined} from 'app/utils';
 
-function getThreadException(thread: Thread, event: Event): ExceptionType | undefined {
+function getThreadException(
+  thread: Thread,
+  event: Event
+): Required<ExceptionType> | undefined {
   const exceptionEntry = event.entries.find(entry => entry.type === 'exception');
 
   if (!exceptionEntry) {
     return undefined;
   }
 
   const exceptionData = exceptionEntry.data as ExceptionType;
-  const exceptionDataValues = exceptionEntry.data.values;
+  const exceptionDataValues = exceptionData.values;
 
   if (!exceptionDataValues?.length) {
     return undefined;
   }
 
   if (exceptionDataValues.length === 1 && !defined(exceptionDataValues[0].threadId)) {
-    return exceptionData;
+    if (!exceptionDataValues[0].stacktrace) {
+      return undefined;
+    }
+    return exceptionData as Required<ExceptionType>;
```

Comment:
Should there be tests for this change in logic?

Reply:
sure thing!

Reply:
done :)